### PR TITLE
CB2-6343 map test codes.

### PR DESCRIPTION
**Pull request by Ian-Hodges**
## Description

VTA has 278 test types, two of which do not map directly to VT. When processing a test result for an NVT or an NVV, bookings are created that can not be rekeyed by the rekey team.

Related issue: [CB2-6343](https://dvsa.atlassian.net/browse/CB2-6343)

## Evidence of Completion
Clean database.

image
PSV record inserted into dynamo table with lbp, nvv, and nvt tests.

image
Three bookings sent to the eventbridge.

image
All three tests have bookings in the database with NVV swapped for NPV and NVT swapped for NPT. LBP remains unchanged.

image

### CODE REVIEW

File: src/util/validators/testCode.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    original: 'NVV',
+    replacement: 'NPV',
+  },
+];
+
+export const validateTestCode = (testCode: string): string => {
+  return (
+    testCodeMaps.find((x) => x.original === testCode)?.replacement ?? testCode
+  );
+};
```
Comment (aaron-downing) [inline src/util/validators/testCode.ts:4]: Something simpler could be
```
const testCodeMaps: {[key: string]: string | undefined} = {
    'NVT': 'NPT',
    'NVV': 'NPV'
}

const validateTestCode = (testCode: string): string => testCodeMaps[testCode] ?? testCode
```

Comment (aaron-downing) [reply src/util/validators/testCode.ts:4]: In this case I wouldn't even wrap `testCodeMaps[testCode] ?? testCode` in a function

Comment (Ian-Hodges) [reply src/util/validators/testCode.ts:4]: I can't decide which way I prefer. Keep changing my mind!

Comment (Ian-Hodges) [reply src/util/validators/testCode.ts:4]: Did it.

**Review by aaron-downing — APPROVED** [review]

**Review by naathanbrown — APPROVED** [review]

**Pull request merged**